### ISSUE TITLE
user warnings about plugins are shown every time when kolibri starts

### ISSUE THREAD

**Issue by lyw07** [learningequality/kolibri#5788]
### Observed behavior

When I started kolibri from a fresh install, I got this warning:
```
/Users/lingyiwang/.venvs/kolibri0.12.6b1/lib/python2.7/site-packages/kolibri/plugins/base.py:87: UserWarning: kolibri.plugins.setup_wizard exposes a KolibriPluginBase derived object but is initialized after Django - enable as a plugin to use this properly
  module=self._module_path()
```
After I created the facility and got navigated to the device page, I got another warning:
```
/Users/lingyiwang/.venvs/kolibri0.12.6b1/lib/python2.7/site-packages/kolibri/plugins/base.py:87: UserWarning: kolibri.plugins.device_management exposes a KolibriPluginBase derived object but is initialized after Django - enable as a plugin to use this properly
  module=self._module_path()
```

### Expected behavior

we should try to eliminate the warnings if possible

### User-facing consequences

users/developers who check the logs might be confused by these warnings

### Errors and logs

```
INFO     ENGINE Serving on http://0.0.0.0:8080
INFO     ENGINE Bus STARTED
/Users/lingyiwang/.venvs/kolibri0.12.6b1/lib/python2.7/site-packages/kolibri/plugins/base.py:87: UserWarning: kolibri.plugins.setup_wizard exposes a KolibriPluginBase derived object but is initialized after Django - enable as a plugin to use this properly
  module=self._module_path()
/Users/lingyiwang/.venvs/kolibri0.12.6b1/lib/python2.7/site-packages/kolibri/plugins/base.py:87: UserWarning: kolibri.plugins.device_management exposes a KolibriPluginBase derived object but is initialized after Django - enable as a plugin to use this properly
  module=self._module_path()
INFO     Downloading data for channel id a9b25ac9814742c883ce1b0579448337
```

### Steps to reproduce

1. get the 0.12.6b1 pex
2. run `python kolibri-0.12.6b1.pex start`
3. open localhost:8080
4. check the log (on Ubuntu, the error appeared in daemon.txt)
5. create the facility
6. check the log again

### Context

Kolibri version: 0.12.6b1
Operating System: OSX, Ubuntu Xenial

**Comment by rtibbles** [comment]
Looking at this, I think this error is completely erroneous. If the initialization was going to cause a problem, it would be raising an error elsewhere, so this is just noise. Going to remove.

**Comment by lyw07** [comment]
fixed in #5792
